.github/.wordlist.txt

```diff
@@ -443,6 +445,7 @@ DoorState
 doru
 DOTBR
 DOVERLAY
+DOGO
```

Comment:
since dogo seems to be a tool name, could we escape it as using backticks and not claim it is a word?